features/meta_data_scenario.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Android support
```

Comment:
All of the handled scenarios should go into a single file and have "Sending handled exceptions" as the feature name.